src/components/dashboard/amend-balance.svelte

```diff
@@ -0,0 +1,62 @@
+<script>
+  import { getContext } from 'svelte';
+  import { TextField, Button } from 'attractions';
+  import { snackbarContextKey } from 'attractions/src/snackbar';
+  import * as api from '@/utils/api.js';
+  import s from '@/utils/plural-s.js';
+
+  const showSnackbar = getContext(snackbarContextKey);
+
+  export let users = [];
+  export let account;
+
+  let amount = '';
+
+  async function send() {
+    try {
+      const promises = users.map(user => api.json(api.patch(`/accounts/${user.email}/balance`, {
```

Comment:
What do you say we add a parameter to the `patch`/`get`/... functions that would automatically call `json` on the response?

Reply:
Nah, I don't think this would help readability. There are no kwargs in JS

Reply:
True, but many use object destructuring as an alternative for that. We already do that, with the object that has `data` and `csrfToken` as properties. Why can't there be a 3rd property `json: true`?

Reply:
Yeah, but this object isn't always passed, and when it is, it doesn't make stuff pretty, so I'd say that a wrapper function is cleaner. Also leads to nicer separation of concerns

Reply:
I just don't like the sight of a function that never gets called without another particular function (or a few), and not having them composed to form one function that does both. Perhaps that's the effect of Haskell on me

Reply:
I do not share your concern, sorry)